instagram/src/components/PostContainer/PostsPage.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import dummyData from "../../dummy-data";
+import PostContainer from "./PostContainer";
+import SearchBar from "../SearchBar/SearchBar";
+ class PostsPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      postData: [],
+      newSearch: "",
+      filtered: false
+    };
+  }
+   componentDidMount() {
+    setTimeout(() => {
+      this.setState({ postData: dummyData });
+    }, 500);
+  }
+   startSearch = event => {
+    event.preventDefault();
+    let newData = dummyData;
```

Comment:
newData should be taken off state, not the original dummyData. What if a post got deleted for example? The post would keep coming back in your filtered data this way, which isn't good.